soroban-env-host/src/events.rs

```diff
@@ -87,13 +87,13 @@ impl Display for DebugArg {
 /// [host::Host::debug_event](crate::host::Host::debug_event) for normal use.
 #[derive(Clone, Debug)]
 pub struct DebugEvent {
-    pub msg: Option<&'static str>,
+    pub msg: Option<Cow<'static, str>>,
```

Comment:
I _think_ this is the right way to do the Cow so it can hold either the static str or the String. I'm not overly familiar with Cow though. @graydon Am I using this right?